### ISSUE TITLE
Recolour shaded table rows in disaggregation report

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1198]
**Please complete the following information**
Data repository URL: All
Data or metadata issue: Data service

Link: https://sdgdata.gov.uk/sdg-data/disaggregations.html

Use GDS suggested grey for shaded rows

**Additional context**
Will help with contrast of text

**Comment by phillipgardiner** [comment]
@brockfanning grey for shaded rows = #f3f2f1

**Comment by brockfanning** [comment]
@phillipgardiner I just wanted to confirm that #f3f2f1 is correct, because it seems very similar - if not identical - to what's already there (or maybe it's my display).

**Comment by phillipgardiner** [comment]
@brockfanning @LucyGwilliamAdmin  

Yes #f3f2f1 is very very similar, however this is the grey specified by GDS so I went with that. However...looking back at the accessibility report I am a bit confused as it states "_The blue (#1D70B8) link text against the grey (rgba(0,0,0,.05)) background, fails colour contrast guidelines with a ratio of 4.6:1._". The original grey used (#F2F2F2) has a contrast ratio of 4.61:1 and the proposed grey (#f3f2f1) has a contrast ration of 4.62:1. 

Update: Ah I see now this one was under 'Contrast (Enhanced) (AAA)' and therefore refers to changes required in order to pass 'AAA'. I think we can just change the grey to (#f3f2f1) in order to align to GDS and keep consistent across our site and this will pass 'AA' contrast rules of at least 4:5:1 as its 4.62.1. @LucyGwilliamAdmin do you agree?

**Comment by brockfanning** [comment]
@phillipgardiner @LucyGwilliamAdmin If it helps to see it in action, https://github.com/open-sdg/sdg-build/pull/262 is ready for review.
